app/ColorPair.js

```diff
@@ -0,0 +1,10 @@
+export class ColorPair {
+    constructor(majorColor, minorColor) {
+      this.majorColor = majorColor;
+      this.minorColor = minorColor;
+    }
+
+    formatColorPair() {

```

Comment:
Good correction of spelling. Consider what a user would like to see: MajorColor? Or Major Color?